reason-mode.el

```diff
@@ -1354,15 +174,17 @@ This is written mainly to be used as `end-of-defun-function' for Reason."
 ;;;###autoload
 (add-to-list 'auto-mode-alist '("\\.rei?\\'" . reason-mode))
 
-
 (defun reason-mode-reload ()
   (interactive)
   (unload-feature 'reason-mode)
+  (unload-feature 'reason-indent)
   (require 'reason-mode)
   (reason-mode))
 
 (require 'refmt)
```

Comment:
Let's keep the `require` on the top. Not that it matters, but I feel it is better to see what it is loaded up there.